cli/src/main/kotlin/commands/ScannerCommand.kt

```diff
@@ -49,20 +49,20 @@ object ScannerCommand : CommandWithHelp() {
         }
     }
 
-    @Parameter(description = "The analyzer result file to use. Source code will be downloaded automatically if " +
-            "needed. This parameter and --input-path are mutually exclusive.",
-            names = ["--analyzer-result-file", "-a"],
+    @Parameter(description = "An ORT result file with an analyzer result to use. Source code will be downloaded " +
+            "automatically if needed. This parameter and '--input-path' are mutually exclusive.",
+            names = ["--ort-file", "-a"],
             order = PARAMETER_ORDER_OPTIONAL)
     private var dependenciesFile: File? = null
 
-    @Parameter(description = "The input directory or file to scan. This parameter and --analyzer-result-file are " +
-            "mutually exclusive.",
+    @Parameter(description = "An input directory or file to scan. This parameter and '--ort-file' are mutually " +
+            "exclusive.",
             names = ["--input-path", "-i"],
             order = PARAMETER_ORDER_OPTIONAL)
     private var inputPath: File? = null
 
-    @Parameter(description = "The list of scopes that shall be scanned. Works only with the " +
-            "--analyzer-result-file parameter. If empty, all scopes are scanned.",
+    @Parameter(description = "The list of scopes whose packages shall be scanned. Works only with the '--ort-file' " +
```

Comment:
out of scope: as we call this 'ORT result file' maybe we could make it consistent with the parameter naming

Reply:
I deliberately do not call the parameter `--ort-result-file` as it's not the result (i.e. output) for the scanner, but its input.